### PR TITLE
Add info commands and isRank param

### PR DESCRIPTION
**Pull request by jaredgrady**

### CODE REVIEW

File: server/commands.js [hunk 1]
```diff
@@ -91,5 +91,39 @@ module.exports = {
...
+  credits(target, room, user) {
+    this.sendHtml('Gryph is brought to you by the following people:<br/>' +
+      '- CreaturePhil - Design and Development<br/>' +
+      '- fender - Development');
+  },
+
+  commands: 'commandlist',
+  comlist: 'commandlist',
+  commandlist(target, room, user) {
+    let comlist = '- /queue: Shows the order for songs to be played.<br/>' +
```
Comment (CreaturePhil) [inline server/commands.js]: use template strings which supports multi line https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Comment (CreaturePhil) [inline server/commands.js]: also I think `cmdList` is a more clear name

**Comment by jaredgrady** [comment]
Got it. I'll make the changes.

**Comment by jaredgrady** [comment]
@CreaturePhil we need to fix addHtml in order to support multiline.

**Pull request merged**